### PR TITLE
Update simple_landscape.myst

### PR DESCRIPTION
**Pull request by DSamuel1**
Description
-----------
Added noiseless curve to the simple landscape tutorial in the docs. (fixed errors in #754 ).

Checklist
-----------

Check off the following once complete (or if not applicable) after opening the PR. The PR will be reviewed once this checklist is complete and all tests are passing.

- [ ] I added unit tests for new code.
- [ ] I used [type hints](https://www.python.org/dev/peps/pep-0484/) in function signatures.
- [ ] I used [Google-style](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html) docstrings for functions.
- [ ] I [updated the documentation](../blob/master/docs/CONTRIBUTING_DOCS.md) where relevant.

If some items remain, you can mark this a [draft pull request](https://github.blog/2019-02-14-introducing-draft-pull-requests/).

Tips
----

- If the validation check fails:

    1. Run `make check-types` (from the root directory of the repository) and fix any [mypy](https://mypy.readthedocs.io/en/stable/) errors.

    2. Run `make check-style` and fix any [flake8](http://flake8.pycqa.org) errors.

    3. Run `make format` to format your code with the [black](https://black.readthedocs.io/en/stable/index.html) autoformatter.

  For more information, check the [Mitiq style guidelines](https://mitiq.readthedocs.io/en/stable/contributing.html#style-guidelines).
  
- Write "Fixes #XYZ" in the description if this PR fixes Issue #XYZ.

### CODE REVIEW

**Comment by andreamari** [comment]
Thanks @DSamuel1!
I have one final comment. The body of the noisy and noiseless executors are very similar. Can we simplify the body of `executor_with_noise`  as follows:
```
    noisy_circuit = circ.with_noise(depolarize(p))
    return noiseless_executor(noisy_circuit)
```
I think it should work.

File: docs/source/examples/simple_landscape.myst [hunk 1]
```diff
@@ -55,20 +55,34 @@ We can visualize the circuit for a particular $\gamma$ as follows.
 print(variational_circuit(gamma=np.pi))
 ```
 
-## Defining the noisy executor
-To interface with Mitiq, we now define an executor function which adds noise to the circuit and computes the
-expectation value of a simple Hamiltonian $H=Z \otimes Z$, i.e., Pauli-$Z$ on each qubit. The code block
-below first creates this observable, then sets a noise value, then defines the executor.
+## Defining the executor functions with and without noise
+To interface with Mitiq, we now define a noiseless executor function, and a executor with noise function which compute the expctation value of a simple Hamiltonian $H=Z \otimes Z$, i.e., Pauli-$Z$ on each qubit without and with noise respectively. The code block below first creates this observable, then defines the noiseless executor, then sets a noise value, then defines the executor with noise function.
```
Comment (rmlarose) [inline docs/source/examples/simple_landscape.myst:59]:
```suggestion
To use error mitigation methods in Mitiq, we define an executor function which computes the expectation value of a simple Hamiltonian $H=Z \otimes Z$, i.e., Pauli-$Z$ on each qubit. To compare to the noiseless result, we define both a noiseless and a noisy executor below.
```

File: docs/source/examples/simple_landscape.myst [hunk 2]
```diff
@@ -94,19 +103,41 @@ The above code block uses depolarizing noise, but any Cirq channel can be substi
 
 +++
 
+## Computing the landscape without noise
+We now compute the energy lanscape $\langle H \rangle(\gamma) =\langle Z \otimes Z \rangle(\gamma)$ with no error in the following code block
```
Comment (rmlarose) [inline docs/source/examples/simple_landscape.myst:107]:
```suggestion
We now compute the energy landscape $\langle H \rangle(\gamma) =\langle Z \otimes Z \rangle(\gamma)$ on the noiseless simulator.
```

File: docs/source/examples/simple_landscape.myst [hunk 3]
```diff
@@ -94,19 +103,41 @@ The above code block uses depolarizing noise, but any Cirq channel can be substi
...
+
+```{code-cell} ipython3
+gammas = np.linspace(0, 2 * np.pi, 50)
+noiseless_expectations = [noiseless_executor(variational_circuit(g)) for g in gammas]
+```
+
+The following code plots the values for visualization
```
Comment (rmlarose) [inline docs/source/examples/simple_landscape.myst:114]:
```suggestion
The following code plots the values for visualization.
```

File: docs/source/examples/simple_landscape.myst [hunk 4]
```diff
@@ -94,19 +103,41 @@ The above code block uses depolarizing noise, but any Cirq channel can be substi
...
+
+```{code-cell} ipython3
+plt.figure(figsize=(8, 6))
+plt.plot(gammas, noiseless_expectations, color="g", linewidth=3, label="Noiseless")
+plt.title(rf"Energy landscape", fontsize=16)
```
Comment (rmlarose) [inline docs/source/examples/simple_landscape.myst:119]:
```suggestion
plt.title("Energy landscape", fontsize=16)
```

**Review by rmlarose — CHANGES_REQUESTED** [review]
Thanks @DSamuel1, looking good modulo typos and wording suggestions. Please see above comments. 

Additionally, I know the following lines weren't changed in this PR but wanted to check:

https://github.com/unitaryfund/mitiq/pull/771/files#diff-695244e8c071fdea1d3c9c0c20662e37f1d5fefc1a1de3bf1c3120244732d78eR184-R194

Is the minimum energy still -1? (The landscape was changed in #742 but I didn't notice this line until now.)

Optional: You can remove the acknowledgements section. Peter contributed this before he joined the team, but since he's a team member we don't need to explicitly acknowledge him anymore.

**Comment by DSamuel1** [comment]
I've added the changes suggested by @rmlarose . As for the minimum energy, I think it should still be -1 based on the last cell?

landscape_cell

**Comment by rmlarose** [comment]
> I've added the changes suggested by @rmlarose . As for the minimum energy, I think it should still be -1 based on the last cell?
> 
> landscape_cell

Yup! Thanks!

**Review by rmlarose — APPROVED** [review]
LGTM! Thanks a lot @DSamuel1 for improving this example!

**Comment by crazy4pi314** [comment]
@all-contributors please add @DSamuel1 for example

**Pull request merged**